desk/app/chat.hoon

```diff
@@ -1693,8 +1711,12 @@
         =.  recency.remark.dm  now.bowl
         =?  cor  &(!=(old-unread di-unread) !=(net.dm %invited))
           (give-unread ship/ship di-unread)
-        ?:  =(our.bowl author.memo)  (di-give-writs-diff diff)
         ?~  entry  (di-give-writs-diff diff)
+        =/  top-con  [id.writ.u.entry time.writ.u.entry]
+        =/  concern  [%reply [id.q.diff now.bowl] top-con]
+        =/  mention  (was-mentioned:utils content.memo our.bowl)
+        =.  di-core  (di-activity concern content.memo mention)
+        ?:  =(our.bowl author.memo)  (di-give-writs-diff diff)
```

Comment:
Why was the "we are the author" conditional moved down (such that we do `+di-activity` unconditionally now)? We still want to not register activity for things we did ourselves, right?

Reply:
because `di-activity` now handles the "self" case by bumping and not sending any activity